lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -64,7 +64,7 @@ class CategoryRenderer {
       const tooltip = this.dom.createChildOf(textEl, 'div', 'lh-error-tooltip-content tooltip');
       tooltip.textContent = audit.result.debugString || 'Report error: no audit information';
     } else if (audit.result.debugString) {
-      const debugStrEl = auditEl.appendChild(this.dom.createElement('div', 'lh-debug'));
+      const debugStrEl = this.dom.createChildOf(titleEl, 'div', 'lh-debug');
```

Comment:
drive by restructuring? seems unrelated

Reply:
Yeah. Kinda separate but hey. I mentioned it on chat yesterday

before:
![image](https://user-images.githubusercontent.com/39191/39677897-b5dec772-5137-11e8-8b86-9881ea1d7db2.png)

after:

![image](https://user-images.githubusercontent.com/39191/39677893-a01f092e-5137-11e8-84bd-923393082700.png)